.github/workflows/build.yml

```diff
@@ -50,22 +45,22 @@ jobs:
     - name: Install Ubuntu debendency
       run: sudo apt-get install -y libgif-dev
 
-    - name: Use Node.js ${{ matrix.node-version }}
+    - name: Use Node.js 10.16
```

Comment:
Does this mean that we'll need to manually update these versions number in future?

Reply:
yes using matrix makes the step on line 38 fail. It's a limit imposed by that helper action.